Template.php

```diff
@@ -418,23 +439,28 @@ protected function get_identifiers_from_url() {
     if (strpos($url, "jstor.org") !== FALSE) {
       if (strpos($url, "sici")) {
         #Skip.  We can't do anything more with the SICI, unfortunately.
+      elseif (strpos($url, "plants")) {
```

Comment:
Can we be confident that the string "plants" would never appear elsewhere in a URL that contains another identifier, e.g. plantscience.com/doi/10.1111/plants.10384?